plugins/opsgenie/plugin.spec.yaml

```diff
@@ -0,0 +1,277 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: opsgenie
+title: OpsGenie
+description: Notify the right people, reduce response time, and avoid alert fatigue within a workflow
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+supported_versions: ["2022-01-11"]
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/opsgenie
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.atlassian.com/pl/software/opsgenie
+tags:
+  - atlassian
+  - opsgenie
+  - alerts
+  - notifications
+hub_tags:
+  use_cases: [alerting_and_notifications]
+  keywords: [atlassian]
+  features: []
+enable_cache: false
+connection:
+  api_key:
+    title: Authorization API Key
+    description: OpsGenie authorization API key
+    type: credential_secret_key
+    required: true
+    example: "1234567e-123c-123c-123c-1234567e9xAd"
+actions:
+  create_alert:
+    title: Create Alert
+    description: Creates an alert for OpsGenie
+    input:
+      message:
+        title: Message
+        description: Message of the alert
+        type: string
+        required: true
+        example: "An example alert message"
+      alias:
+        title: Alias
+        description: Client-defined identifier of the alert, that is also the key element of Alert deduplication
+        type: string
+        required: false
+        example: "An example alias"
+      description:
+        title: Description
+        description: Description field of the alert that is generally used to provide a detailed information about the alert
+        type: string
+        required: false
+        example: "An example description"
+      responders:
+        title: Responders
+        description: 'Teams, users, escalations and schedules that the alert will be routed to send notifications. "id/name": Either id or name of each responder should be provided. "type": team, user, escalation, schedule. Format: [{"id/name":"value", "type":"team/user/escalation/schedule"}]'
+        type: "[]object"
+        required: false
+        example: '[{"id":"4513b7ea-3b91-438f-b7e4-e3e54af9147c", "type":"team"},{"name":"NOC","type":"team"}]'
```

Comment:
This is going to be too difficult to use in-product as it is a JSON object input. I would split this into three separate input fields.